docs/_articles/en/lwc/lwc-get-started.md

```diff
@@ -0,0 +1,100 @@
+
+Before you start, install [Salesforce Extensions for Desktop](./en/vscode-desktop/install) or [Set Up Code Builder](./en/codebuilder/cb-setup)
+
+### Add an Apex Class that Queries Contacts
+1. In VS Code, run the command **SFDX: Create Project** to create a Salesforce DX project if you don’t have one.
+2. Log in to the org to which you want to add the LWC.
+3. Run the **SFDX: Refresh SObject Definitions** command from the Command Palette to get completion suggestions for your SObjects related code.
+4. From the command palette run **SFDX: Create Apex Class** and create a class called `ContactController`.
+
+Next let's add a method called `getContacts` to this class. We’ll make this method both public and static. Because it’s a static method, you don't need to create an instance of the class to access the method—you can just use the name of the class followed by a dot (`.`) and the name of the method. This method queries a set of fields on the contact object. Paste this code into `contactController.cls`:
+
+
+```
+public with sharing class ContactController {
+    @AuraEnabled(cacheable=true)
+    public static List<Contact> getContacts() {
+        return [
+SELECT 
+Id, 
+Name, 
+Email, 
+Phone 
+FROM Contact 
+WITH SECURITY_ENFORCED 
+LIMIT 10
+];
+    }
+}
+
+```
+Next we'll add the LWC component that displays queried fields.
+
+### Create an LWC Component that Displays Contact Information
+1. From the command palette run **SFDX: Create Lightning Web Component** and create a component called `contactList` in the default location.
+2. In the `contactList.html file`, cut and paste the following code and save the file:
+
+```
+<template>
+    <lightning-card title="Contact List">
+        <template if:true={contacts.data}>
+            <template for:each={contacts.data} for:item="contact">
+                <div class="slds-var-p-horizontal_medium" key={contact.Id}>
+                    <p>{contact.Name}, {contact.Phone}</p>
+                </div>
+            </template>
+        </template>
+        <template if:true={contacts.error}>
+            <p>{contacts.error}</p>
+        </template>
+    </lightning-card>
+</template>
+
+```
+3. In the contactList.js file, cut and paste this code and save:
+
+```
+import { LightningElement, wire } from 'lwc';
+import getContacts from '@salesforce/apex/ContactController.getContacts';
+
+export default class ContactList extends LightningElement {
+    @wire(getContacts) contacts;
+}
+```
+4. Cut and paste the following code in the contactList.js-meta.xml file and save:
+   
+```
+ <?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+ 	<apiVersion>57.0</apiVersion>
+ 	<isExposed>true</isExposed>
+ 	<targets>
+
+   		<target>lightning__HomePage</target>
+ 	</targets>
+</LightningComponentBundle>
+  
+```
+5. Right-click the default folder under `force-app/main` and run **SFDX: Deploy Source to Org** to deploy your code to the org.
```

Comment:
by following the instructions I cannot succeed in deploying the code to the org. Here are the errors I got:

![image](https://user-images.githubusercontent.com/132491513/236892719-8f263abb-3d95-4558-ac47-ff2bf9ed6f30.png)